### PR TITLE
[FR] Refactor events

### PR DESCRIPTION
**Pull request by lbourdois**
@lewtun 
Everything is in the title :)

### CODE REVIEW

File: chapters/fr/_toctree.yml [hunk 1]
```diff
@@ -191,7 +191,11 @@
     title: Quiz de fin de chapitre
     quiz: 9
 
-- title: Evènements liés au cours d'Hugging Face
+- title: Evènements liés au cours
   sections:
   - local: event/1
```
Comment (lewtun) [inline chapters/fr/_toctree.yml:196]: turbo nit: I renamed this folder to `events` in #329 - would you mind doing the same please?

**Review by lewtun — APPROVED** [review]
This looks great, thanks for refactoring the events @lbourdois ! I've left one nit to address, but then we can merge :)

File: chapters/fr/_toctree.yml (hunk 1 above)
Comment (lbourdois) [reply chapters/fr/_toctree.yml]: @lewtun  Done

**Comment by lewtun** [comment]
Thanks for the tweak!

**Pull request merged**